src/parsec.rs

```
@@ -766,10 +766,40 @@ impl<T: NetworkEvent, S: SecretId> Parsec<T, S> {
             }).collect()
     }
 
-    // Initialise the membership list of the creator of the given event.
-    // Do nothing if already initialised.
-    fn initialise_membership_list(&mut self, _event_hash: &Hash) {
-        // TODO
+    // Initialise the membership list of the creator of the given event to the same membership list
+    // the creator of the other-parent had at the time of the other-parent's creation. Do nothing if
+    // the event is not a request or response or if the membership list if already initialised.
+    fn initialise_membership_list(&mut self, event_hash: &Hash) {
+        let (creator, other_parent_creator, other_parent_index) = {
+            let event = if let Some(event) = self.get_known_event(event_hash).ok() {
```

Do we really want to swallow this error?